modules/classrooms/classrooms.go

```diff
@@ -135,40 +106,108 @@ func GetClassrooms(w http.ResponseWriter, r *http.Request, collection *mongo.Col
 	if err := cursor.Err(); err != nil {
 		// If there is an error iterating through the cursor,
 		// log the error and return an internal server error response
-		fmt.Println("Error iterating cursor:", err)
-		http.Error(w, err.Error(), http.StatusInternalServerError)
+		logger.Error(fmt.Errorf("Error iterating cursor: " + err.Error()))
+		http.Error(w, "Error iterating cursor.", http.StatusInternalServerError)
 		return
 	}
 
 	w.WriteHeader(http.StatusOK)
 	json.NewEncoder(w).Encode(classrooms)
```

Comment:
Should we log when the function when it finishes? Might be helpful later while debugging issues.

Reply:
Sure, I will add a line of code logging info to the end of each function and comment it out so that the developer can uncomment if they need to debug anything